packages/fx-core/src/plugins/resource/function/resources/errors.ts

```diff
@@ -243,6 +243,12 @@ export class PublishCredentialError extends FunctionPluginError {
   }
 }
 
+export class FindAppError extends FunctionPluginError {
+  constructor() {
+    super(ErrorType.User, "FindAppError", "Failed to find the function app.", [tips.doProvision]);
```

Comment:
I think this could be system error. We just successfully created the function app in provision stage, and should be able to find it in post-provision stage, otherwise there may be a bug.

Reply:
It is fixed. Thanks.